packages/react/src/components/Breadcrumb/BreadcrumbItem.js

```diff
@@ -63,11 +63,17 @@ const BreadcrumbItem = React.forwardRef(function BreadcrumbItem(
     );
   }
 
+  const customChildClassName = children?.props?.className;
+  const childClassName = cx({
+    [`${prefix}--link`]: true,
+    [customChildClassName]: !!customChildClassName,
+  });
+
   return (
     <li className={className} ref={ref} {...rest}>
       {React.cloneElement(children, {
         'aria-current': ariaCurrent,
-        className: `${prefix}--link`,
+        className: childClassName,
```

Comment:
The classnames package won't include values for a key that is falsey, so I think the object notation variables above can be removed and just a small change to this single line:
```suggestion
        className: cx(`${prefix}--link`, children.props.className),
```